googlechromelabs/ise-web-security-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+	"bundles": {
```

Comment:
Oh, similar mistake as in the yaml file, here you mixed 2 and 4 spaces, but should be 4 everywhere :thinking: 
Didn't see that just now, sorry for the double work :see_no_evil: